src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 //
+using System;
```

Comment:
Is this change necessary, as I don't see other changes to the file?

Reply:
Good catch, no not anymore. I'll remove it.